### ISSUE TITLE
AttributeError: 'MultivariateNormalTriL' object has no attribute 'mu' normal_sghmc.py

### ISSUE THREAD

**Issue by philipperemy** [blei-lab/edward#739]
I have this error when running `python3 normal_sghmc.py`.

```
Inferred posterior mean:
[ 0.7199108   0.68983078]
Inferred posterior stddev:
[ 1.60923088  1.58413029]
Traceback (most recent call last):
  File "normal_sghmc.py", line 66, in <module>
    mvn_plot_contours(z, ax=ax)
  File "normal_sghmc.py", line 24, in mvn_plot_contours
    mu = sess.run(z.mu)
AttributeError: 'MultivariateNormalTriL' object has no attribute 'mu'
```

Should it be replaced by?
```
mu = sess.run(z.parameters['loc'])
Sigma = sess.run(z.parameters['scale_tril'])
```

**Comment by dustinvtran** [comment]
Good catch. Yes. PR welcome

**Comment by dustinvtran** [comment]
Fixed in #740
